src/components/Select/InternalComponents/MultiSelectSearchable.tsx

```diff
@@ -43,6 +43,10 @@ export const MultiSelectSearchable = () => {
 		getToggleButtonProps();
 	const { id, onKeyDown, ...restInputProps } = getInputProps();
 
+	if (restInputProps.onFocus) {
+		restInputProps.onFocus = undefined; // Override downshift behavior to fix NEO-2175
+	}
```

Comment:
nit: prefer one liner:
```suggestion
	delete restInputProps.onFocus; // remove downshift behavior to fix NEO-2175
```

Reply:
same in `SingleSelectSearchable.tsx`

Reply:
yes, that was my original implementation but biome flagged using delete as a performance issue and suggested I do the assignment to undefined. Thoughts?

Reply:
[This codesandbox](https://codesandbox.io/p/sandbox/usecombobox-action-props-z6qkg?file=%2Fsrc%2Findex.js)  shows how to override onFocus().

Reply:
@yangAtSpoken @joe-s-avaya  Yes Yang, that would be another way of doing it.  In our case we don't want to pass a new implementation but simply delete the one from downshift.  Using the codesandbox pattern my code would look something like this:
onFocus: () => undefined
or 
onFocus: () => {}

Any thoughts?

Reply:
Does it help if you call openMenu() in onFocus?  I would like to keep the current behavior when it gets the keyboard focus.

Reply:
> the latter (`onFocus: () => {}`)

using the empty object causes a warning so I'll leave it as undefined.

Reply:
> Does it help if you call openMenu() in onFocus? I would like to keep the current behavior when it gets the keyboard focus.

I tried the openMenu() solution but it does not fix the bug. So far overriding with undefined seems to be the best solution.